sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -2,14 +2,17 @@
 
 ## 2.51.0-beta.1 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
+- The details that query used to fetch are now ingested directly in Customer's log Analytics workspace and hence query operation is no more valid on connection Monitor resource now.
```

Comment:
- Removed `query` from `ConnectionMonitor`. Its functionality is now provided by Log Analytics service.